tests/wpunit/PMPro_SubscriptionTest.php

```diff
@@ -50,24 +50,54 @@ public function test___construct_with_id() {
 	public function test___construct_with_array() {
 		// Basic test data, just enough to test with.
 		$subscription_data = [
-			'id'                  => '1234',
-			'user_id'             => '2345',
-			'membership_level_id' => '3456',
-			'status'              => 'active',
+			'id'                          => '1234',
+			'user_id'                     => '2345',
+			'membership_level_id'         => '3456',
+			'status'                      => 'active',
+			'gateway'                     => 'stripe',
```

Comment:
Let's keep this test simplified and not focused on all of the args at the moment. The test__call method will cover things more thoroughly and this just ends up covering the same things as that at when all the extra args are used/checked.